about.md

```diff
@@ -0,0 +1,217 @@
+
+```
+## Markdown Heading
+
+This is an example of using Markdown to format your document.  It's not very difficult.
+
+* Here are some bullet points.
+* Markdown is pretty simple to understand...
+  * But it's also really readable.
+  * HTML can't even compare!
+* You'll get the hang of it in no time at all.
+```
+
+### Control Your Metadata With YAML
+
+As we covered earlier, all of Pico's content is stored in flat files.  This includes the Metadata for your pages as well.
+
+Metadata makes up all the little details that accompany your page.  Things like the title, author, description, post date, etc.  While other CMS's would store this data in the database with everything else, Pico simply places it at the beginning of the file.
+
+Each Markdown-based page you create in Pico will have a small header at the top that contains all the metadata for that file.  The metadata is formatted using [YAML][YAML], which makes it really easy to manage.  We go more in-depth about [using YAML][DocsYAML] for your page header in our Documentation.
+
+Let's say for instance that you have a page that you'd like to back-date to last January.  Maybe that was the date you started your project or organization and you want the page to reflect that.  All you do is... type in that date!  There's no hoops to jump through, you don't have to go into some management console to modify your post, you just change the date.  You also don't have to worry that that minor edit you made to an older file will silently change a page's date to today's, thoroughly confusing all your visitors.
+
+```
+---
+Title: YAML Example
+Description: This is what your Metadata will look like in YAML
+Author: Your Name
+Date: 2016/05/10
+---
+```
+
+## Flexibility and Customization
+
+Pico is not just easy to use, it's *powerful*!  Pico has two very large avenues of customization, [Themes and Plugins][Customization].
+
+### Theming With Twig Templates
+
+Pico's default theme is *not* intended for production use.  It is provided as a great, but minimal starting place for you to develop your own customized website.  If you aren't familiar with HTML, fear not, we have an ever-growing variety of community-created themes available [here on our site][SiteThemes], as well as some more on [our wiki][WikiThemes].
+
+Pico's "Themes" are built using [Twig Templates][Twig].  Twig is a template engine which provides an easy and powerful way to customize your website.  You can use small amounts of Twig to add dynamic content to a mostly-static HTML website, or use large amounts of Twig to build in some really incredible features.  Twig is so powerful that you'll find it can accomplish most tasks by itself, eliminating the need for plugins.  You can find more information on making your own templates in [Twig's Documentation][TwigDocs] and see how they relate to Pico in our own [Documentation][DocsThemes].
```

Comment:
I think that was a relic of a previous version.  I think I started it as `Pico's "Themes" are actually Twig Templates` but changed the wording so it would sound better.  I'll remove them.